pyvista/core/filters/data_set.py

```diff
@@ -596,7 +596,7 @@ def slice_along_line(dataset, line, generate_triangles=False,
 
         Examples
         --------
-        Slice the random hills dataset along a circular arc.
+        Slice along a circular arc.
```

Comment:
I'd prefer the original version here too.

Reply:
I think we ended up with a merge conflict here and I overwrote it with the original.

Reply:
Reverted in 55ba3daa9ec943acb388ac83608d7bd7ca5341b1.